_includes/v3.10/release-notes/v3.10.0-release-notes.md

```diff
@@ -0,0 +1,59 @@
+ - Support datastore-driven clusterIP updates. [confd # 272](https://github.com/projectcalico/confd/pull/272) (@caseydavenport)
+
+#### Bug fixes
+
+ - Fix returning of resp.Body.Close() error. [node #329](https://github.com/projectcalico/node/pull/329) (@xieyanker)
+ - Fix erroneous "Partial success" output. [calicoctl #2070](https://github.com/projectcalico/calicoctl/pull/2070) (@lmm)
+ - Fix logging to be independent of go mod versions. [libcalico-go #1127](https://github.com/projectcalico/libcalico-go/pull/1127) (@neiljerram)
+
+#### Other changes
+
+ - Update calico/node to use a version of calico/bird based on upstream BIRD v1.6.7. This allows us to make use of LLGR (long-lived graceful restart). [node #314](https://github.com/projectcalico/node/pull/314) (@neiljerram)
+ - Use a real confd version. [node #315](https://github.com/projectcalico/node/pull/315) (@rafaelvanoni)
+ - Add semaphore config. [node #324](https://github.com/projectcalico/node/pull/324) (@neiljerram)
+ - The default IP pool block size is now configurable. Use the `CALICO_IPV4POOL_BLOCK_SIZE` and `CALICO_IPV6POOL_BLOCK_SIZE` environment variables to configure the default pool block size. [node #346](https://github.com/projectcalico/node/pull/346) (@halfcrazy)
+ - The `CALICO_IPV4POOL_NODE_SELECTOR` and `CALICO_IPV6POOL_NODE_SELECTOR` environment variables were added for configuring default pool node selectors. [node #346](https://github.com/projectcalico/node/pull/346) (@tmjd)
+ - Add calicoctl `patch` command similar to `kubectl patch`. [calicoctl #2054](https://github.com/projectcalico/calicoctl/pull/2054) (@syscll)
+ - Update calicoctl alpine image to 3.10. [calicoctl #2061](https://github.com/projectcalico/calicoctl/pull/2061) (@chenrui333)
+ - Re-enable golangci-lint linters and fix lint warnings. [felix #2130](https://github.com/projectcalico/felix/pull/2130) (@rafaelvanoni)
+ - Add support to watch stderr for FV containers. [felix #2137](https://github.com/projectcalico/felix/pull/2137)(@song-jiang)
+ - Reduce severity of BPF log levels. [felix #2144](https://github.com/projectcalico/felix/pull/2144) (@tmjd)
+ - Fix felix log level. [felix #2147](https://github.com/projectcalico/felix/pull/2147) (@neiljerram)
+ - Clean up code. [felix #2150](https://github.com/projectcalico/felix/pull/2150) (@beautytiger)
+ - Run typha as non-root user. [typha #312](https://github.com/projectcalico/typha/pull/312)
+ - Improve log for check command failing. [confd #271](https://github.com/projectcalico/confd/pull/271) (@neiljerram)
+ - Move some testutils functions to utils_linux.go. [cni-plugin #787](https://github.com/projectcalico/cni-plugin/pull/787) (@song-jiang)
+ - Use 'git diff' instead of 'git diff-index' in Makefile's git-commit target. [cni-plugin #788](https://github.com/projectcalico/cni-plugin/pull/788) (@rafaelvanoni)
```

Comment:
This and the one below aren't user-facing so we should probably remove from here.